data/sample/amex.csv

```diff
@@ -0,0 +1,10 @@
+Date,Description,Amount,Extended Details,Appears On Your Statement As,Address,City/State,Zip Code,Country,Reference,Category

```

Comment:
this is what the amex csv actually looks like, 10 lines for 4 actual lines of data because of their quoting and newlines within fields 🤦